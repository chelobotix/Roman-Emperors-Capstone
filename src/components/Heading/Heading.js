import style from "./Heading.module.css";

const Heading = () => <div className={style.heading}></div>;

export default Heading;
