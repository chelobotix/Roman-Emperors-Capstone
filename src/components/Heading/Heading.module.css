.heading {
  width: 100%;
  height: 300px;
  background-image: url("../../assets/images/headingImg.svg");
  background-repeat: no-repeat;
  background-attachment: fixed;
  background-position: center -39px;
}
