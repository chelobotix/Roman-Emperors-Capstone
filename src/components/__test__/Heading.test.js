import '@testing-library/jest-dom/extend-expect';
import renderer from 'react-test-renderer';
import Heading from '../Heading/Heading';

describe('Header', () => {
  test('Header Tree', () => {
    const tree = renderer.create(
      <Heading />,
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
