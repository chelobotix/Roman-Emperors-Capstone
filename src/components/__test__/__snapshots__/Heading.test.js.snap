// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Header Tree 1`] = `
<div
  className="heading"
/>
`;
